src/remotestorage.js

```diff
@@ -341,13 +341,20 @@
         this.remote.configure(info);
         if (! this.remote.connected) {
           if (info.authURL) {
-            this.authorize(info.authURL);
+            if (typeof token === 'undefined') {
```

Comment:
Hmm. The overwhelming amount of cases is the first one, though. That will be the case for more than 99.99% of calls. So that's why I thought the skipping should be the `else`. Maybe with this explanation it seems more reasonable to you as well?